packages/platform/atoms/booker/BookerWebWrapper.tsx

```diff
@@ -58,6 +59,7 @@ export const BookerWebWrapper = (props: BookerWebWrapperAtomProps) => {
     bookingUid: bookingUid,
     layout: bookerLayout.defaultLayout,
     org: props.entity.orgSlug,
+    timeZone,
```

Comment:
I don't think I fully understand the reasoning of this timezone property on booker store. Shouldn't we just update the timzone in the timePreferencesStore?

Reply:
This is causing us to handle a new place to store the timezone

Reply:
Tested it out and it seems to make sense now. The idea is this if I understand correctly:
- 'cal.tz' query param has the highest priority and if available it skips user's preferred timezone in localStorage.
- If not available, it should keep on using the user's preferred timezone.
- It won't be wise to reset the user's preference(in localStorage) through this query param.